builtin-gconf.c

```diff
@@ -1038,6 +1040,34 @@ gconf_entry_get_value(const GConfEntry *entry)
   return entry ? entry->value : 0;
 }
 
+/** Get reset default value for config entry
+ */
+static const char *gconf_entry_get_default(const GConfEntry *entry)
+{
+  /* Brightness values are a special because the config
+   * defaults use legacy 1-5 range and are migrated to
```

Comment:
Hm, this looks ugly. Doesn't it know whether or not it has already migrated the data, so that we can - in mce's source code - just place all the new values, and when mce is set up for the first time, the config is written "in the new format" (with the newer version number), but when an old config is opened, it's migrated (values converted + new format version number written into config).

IMHO this whole function is just a workaround for what should be handled on the settings side.